pkg/roachprod/vm/azure/testdata/startup_script

```diff
@@ -207,6 +207,7 @@ sudo sh -c 'echo "PubkeyAcceptedAlgorithms +ssh-rsa" >> /etc/ssh/sshd_config'
 
 
 sudo sed -i 's/#LoginGraceTime .*/LoginGraceTime 0/g' /etc/ssh/sshd_config
+sudo sed -i 's/TCPKeepAlive no/TCPKeepAlive yes/g' /etc/ssh/sshd_config
```

Comment:
Afaik, it defaults to "yes" unless explicitly set. I guess under IBM's linux it may flip to "no"?

Reply:
Yeah, `yes` is the default, and it is switched to `no` on IBM's Ubuntu images.